lua/ttt2/libraries/game_effects.lua

```diff
@@ -148,6 +148,75 @@ function gameEffects.RadiusDamage(dmginfo, pos, radius, inflictor)
     end
 end
 
+-- Creates explosion damage in a sphere through walls. Very useful for making explosives that aren't line of sight based.
+-- @param Player dmgowner The player that causes this explosion.
```

Comment:
this should be `attacker` according to [the wiki](https://wiki.facepunch.com/gmod/CTakeDamageInfo)